src/client/client.ts

```diff
@@ -73,11 +73,30 @@ export class Client extends BaseClient {
   private async initClient(mainResponse: string, symbol: string): Promise<boolean> {
     this.symbol = symbol;
     const $ = cheerio.load(mainResponse);
-    // REMOVE LINE BELOW WHEN CREATING THIS FEATURE
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    const links: string[] = $('.panel.linkownia.pracownik.klient a[href*="uonetplus-uczen"]').map(
+    this.urlList = $('.panel.linkownia.pracownik.klient a[href*="uonetplus-uczen"]').map(
       (index, value) => $(value).attr('href'),
     ).get();
     return true;
   }
+
+  public async getDiaryList(): Promise<any[]> {
+    const results = [];
+    this.urlList.forEach((url) => {
+      results.push(this.post(
+        `${url}UczenDziennik.mvc/Get`,
+        {},
+      ));
+    });
+    const responses = await Promise.all(results);
+    responses.forEach((response) => {
+      if (response.data.success === true) {
+        for (let i = 0; i < response.data.data.length; i += 1) {
+          response.data.data[i].url = response.config.url
+            .substring(0, response.config.url.length - 22);
+          response.data.data[i].host = this.host;
+        }
+      }
+    });
+    return [].concat(...responses.map((x) => x.data.data));
```

Comment:
Wouldn't it be simpler?
```suggestion
    responses
      .filter((response) => response.data.success === true)
      .flatMap((response) => // : Array<UserObject>
        response.data.data.map((responseData) => ({ // : UserObject
          IdDziennik: responseData.IdDziennik,
          IdUczen: responseData.IdUczen,
          url: response.config.url.substring(0, response.config.url.length - 22),
          host: this.host,
        }))
      );
    return responses
```